### ISSUE TITLE
can Quod Libet offer a featuren which user can see lyrics on the top of screen like other online music app ?

### ISSUE THREAD

**Issue by karhow** [quodlibet/quodlibet#4466]
The proposed change
-------------------

Benefits of this change
-----------------------

People can see lyrics when doing other things with the computer. 
![image](https://github.com/quodlibet/quodlibet/assets/32405381/b2a3f093-d729-49fa-929c-547b8a567550)

Any issues this might cause
---------------------------

None

**Comment by declension** [comment]
Have you tried any of the lyrics plugins?

**Comment by karhow** [comment]
> Have you tried any of the lyrics plugins?

Yes! I use both two lyrics plugins in the app. However, it doesn't fit for me. I want to show the lyrics on the top of all other apps, even I didn't keep Quod Libet on the top of screen. Had you used other online music app like 'qq music' which have the feature that show the lyrics on the top of all apps, through it I can see lyrics when I am working and I didn't need to keep Quod Libet in my filed of vision.
